core/controllers/suggestion.py

```diff
@@ -485,11 +485,24 @@ def _construct_exploration_suggestions(suggestions):
 
     suggestion_dicts = []
     for suggestion in suggestions:
-        content_html = exp_id_to_exp[suggestion.target_id].get_content_html(
-            suggestion.change.state_name, suggestion.change.content_id)
-        suggestion_dict = suggestion.to_dict()
-        suggestion_dict['exploration_content_html'] = content_html
-        suggestion_dicts.append(suggestion_dict)
+        available_states = exp_id_to_exp[suggestion.target_id].states
+        is_content_id_available = False
```

Comment:
content_id_is_available (don't start variable names with verbs).

Or better: content_id_exists (not sure what you mean by "available", it could mean there's a free slot there that can be taken up).

Reply:
Done